Makefile

```diff
@@ -72,13 +66,12 @@ test_check_headers:
 
 .PHONY: check_headers
 check_headers:
-	@echo "Checking file headers"
-	@python test/verify_boilerplate.py
+	@source test/make.sh && check_headers
 
 # Integration tests
 .PHONY: test_integration
 test_integration:
-	./test/ci_integration.sh
+	test/ci_integration.sh
```

Comment:
Any particular reason this `./` prefix was removed?

Reply:
Removed due to lack of need

Reply:
I think we should prefer the explicit path reference (`./`) here.

Reply:
Ok, no problem